packages/ui/src/components/va-badge/hooks/useFloatingPositionStyles.ts

```diff
@@ -15,9 +17,13 @@ export const useFloatingPositionProps = {
     validator: (position: PlacementWithAlias) => placementsPositionsWithAliases.includes(position),
   },
   offset: {
-    type: [Number, String] as PropType<string | number>,
+    type: [Number, String, Array] as PropType<BadgeOffsetProp>,
```

Comment:
Offset prop definitely needs more explanation.

Right now there is no examples even.